packages/lib/services/environment.ts

```diff
@@ -76,7 +78,94 @@ export const getEnvironments = cache(async (productId: string): Promise<TEnviron
   }
 });
 
-export const updateEnvironment = async (environmentId: string, data: Partial<TEnvironmentUpdateInput>): Promise<TEnvironment> => {
+export const getEnvironmentBySession = cache(async (user: any): Promise<any> => {
```

Comment:
this service isn't the best approach:
- a service should take an ID as attribute (e.g. userId) not on object
- it's not clear what it returns as a user is member of multiple teams, that have multiple products that have multiple environments: what environment is returned
- a membership is created in a get method. that is not expected by the user.

Reply:
Removed this service, as we don't need it anymore :)